CONTRIBUTING.md

```diff
@@ -41,6 +44,27 @@ guidelines below.
 
 ## Getting started
 
+### Local development
+
+You can serve this site locally to quickly see your changes and additions before you PR them. To get started, navigate into your new site’s directory and start it up, as follows.
+
+```shell
+cd docs-website/
+yarn
+yarn start
+```
+
+Your site is now running at `http://localhost:8000`!
+
+### Dependencies
+
+Node v12 is used in this project as specified in [.nvmrc](https://github.com/newrelic/developer-website/blob/master/.nvmrc).
+
+## Unit tests
```

Comment:
It looks like the following set of content is considered a child of this `Unit tests` section (see the ToC). I'm not sure how best to update this, but could you see if you can make sure this is organized so that this unit tests section is at the same level as the "using multiple versions of node", etc.?

Reply:
Yeah I can fix, it's just the Markdown Header (h1, h2, h3) that sets that in the TOC.